migrations/versions/cdbd609e2bd5_.py

```diff
@@ -0,0 +1,37 @@
+"""
+1.3.1 - Rename biospecimen table column volume_ml to volume_ul
+
+Revision ID: cdbd609e2bd5
+Revises: ce01ada6ac80
+Create Date: 2018-08-27 10:28:59.687358
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import dataservice
```

Comment:
Doesn't seem to be used anywhere

Reply:
use this import while upgrading database